src/services/completions.ts

```diff
@@ -2370,6 +2370,31 @@ namespace ts.Completions {
                     return isPropertyDeclaration(contextToken.parent);
             }
 
+            // If `constructor` is totally not present, but we request a completion manually at a space...
+            if (contextToken === previousToken && isPreviousPropertyDeclarationTerminated(contextToken, position)) {
+                return false; // Don't block completions.

```

Comment:
Do we even know that the current location is in a class right here?

Reply:
Agree. Narrowing this fix to be inside a `ClassLike` should be more reliable.